tools/subgraph-example/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "hedera-subgraph-greeter-example",
```

I may have missed the `@` in the previous comment
```suggestion
  "name": "@hedera-subgraph-greeter-example",
```